spec/javascripts/report-a-problem-spec.js

```diff
@@ -14,11 +14,20 @@ describe("form submission for reporting a problem", function () {
 
   describe("clicking on the toggle", function(){
     it("should toggle the visibility of the form", function() {
+      spyOn(GOVUK.analytics, "trackEvent");
       expect($form).toBeVisible();
+
       $('.js-report-a-problem-toggle').click();
       expect($form).toBeHidden();
+      expect(GOVUK.analytics.trackEvent).toHaveBeenCalledWith(
```

Comment:
Same here. I would create a unit test for the event that is sent when a form is closed.